lib/ngx/balancer_stream.lua

```diff
@@ -0,0 +1,82 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```

Comment:
Or perhaps ngx.stream.balancer and ngx.http.balancer ?

That way the namespace expands with other APIs ported.